packages/plazo_mx/domain/lib/features/login/repository/handle_data.dart

```diff
@@ -0,0 +1,42 @@
+import 'dart:async';
+
+abstract class Validation {
```

Comment:
need to refactor this logic. in my view, u should have the validation model, like

LoginValidationSchema
|
|-loginValidators
|
|-passwordValidators

loginValidators and passwordValidators are list of Validator

Validator - abstract class which have validate method, which returns true (if valid) and false(if invalid).
Validator has several subclasses, for ex. RegExpValidator, MinLengthValidator, RequiredValidator. Each of them has own implementation of validate method. 

And for Validation u need to create separate usecase, for ex. LoginValidatorUseCase

LoginValidator will iterate over login and password validators lists and return a validation result